controller/app_controller.py

```diff
@@ -0,0 +1,44 @@
+from model.repository.users_credentials_repository import UsersCredentialsRepository
+from model.repository.users_repository import UsersRepository
+from utils.db import Base, engine
+
+
+class AppController:
+    def __init__(self, users_repository):
+        self.users_repository = users_repository
+
+    def create_user(self, user_id, first_name, last_name, email, address, phone_number):
```

Comment:
Are we ever going to use this? We use add_user all the time, no? We'll never create a user without a username and password